addons/mail/static/src/components/discuss_sidebar_category/discuss_sidebar_category.scss

```diff
@@ -1,95 +1,12 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarCategory_addingItemInput {
-    margin-left: $o-mail-discuss-sidebar-category-item-avatar-left-margin;
-    margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-}
-
-.o_DiscussSidebarCategory_command {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right:$o-mail-discuss-sidebar-category-item-margin;
-
-    &:first-child {
-        margin-left: 0;
-    }
-
-    &:last-child {
-        margin-right: 0;
-    }
-}
-
-.o_DiscussSidebarCategory_commands {
-    display: flex;
-}
-
-.o_DiscussSidebarCategory_header {
-    display: flex;
-    align-items: center;
-    margin: 5px 0;
-}
-
-.o_DiscussSidebarCategory_headerItem {
-    margin-left: $o-mail-discuss-sidebar-category-item-margin;
-    margin-right: $o-mail-discuss-sidebar-category-item-margin;
-
-    &:last-child {
-        margin-right: $o-mail-discuss-sidebar-scrollbar-width;
-    }
-}
-
-.o_DiscussSidebarCategory_title {
-    display: flex;
-    align-items: baseline;
-}
-
 .o_DiscussSidebarCategory_titleIcon {
     width: $o-mail-discuss-sidebar-category-title-icon-size;
     height: $o-mail-discuss-sidebar-category-title-icon-size;
 }
 
-
-// ------------------------------------------------------------------
-// style
-// ------------------------------------------------------------------
-
-.o_DiscussSidebarCategory_addingItemInput {
-    outline: none;
-}
-
-.o_DiscussSidebarCategory_command {
-    cursor: pointer;
-
-    &:not(:hover) {
-        color: gray('600');
-    }
-}
-
-.o_DiscussSidebarCategory_counter {
-    background-color: $o-brand-primary;
-    color: gray('300');
-}
-
 .o_DiscussSidebarCategory_headerItem {
     user-select: none;
 }
 
-.o_DiscussSidebarCategory_title {
-    cursor: pointer;
-
-    &:not(:hover)
-    {
-        color: gray('600');
-    }
-}
-
 .o_DiscussSidebarCategory_titleIcon {
     font-size: 0.75em;
```

Comment:
Maybe `.small` instead ? :)

Reply:
I like this solution, but I notice that the `DIRECT MESSAGES` item moves up and down when clicked after applying this class... 🤔 

What do you think @stefanorigano ?

Reply:
Finally, we talked about this yesterday and we decided we could use `small`. 👍